tests/test_config.py

```diff
@@ -17,17 +17,30 @@ def temp_path():
         yield Path(td, "file")
 
 
-def test_load_from_object(app):
-    class Config:
-        not_for_config = "should not be used"
-        CONFIG_VALUE = "should be used"
+class Config:
```

Comment:
Can you please rename this class from `Config` to something else? This line is overriding the import done by `from sanic.config import Config`

Reply:
@harshanarayana i renamed the class, thank.

Reply:
Renaming this class seems to be the only thing holding it up.